src/slave/containerizer/mesos/paths.hpp

```diff
@@ -320,6 +320,17 @@ std::string container(
     const ContainerID& containerId,
     bool leaf = false);
 
+
+// Get a containerId from a container's cgroup.
+// Inverse of `cgroups2::container(root, id)`.
+//
+// The provided cgroup can be an absolute path or a relative path.
+// Leaf paths (which end in `/leaf`) and non-leaf paths will resolve to the
+// same container id.
```

Comment:
to be consistent with our other "cgroup" arguments and variables in the cgroups2 library, let's only support the relative path format here